pkgs/flox-pkgdb/default.nix

```diff
@@ -83,6 +83,24 @@
       name = "set-prompt.zsh.sh";
       path = ../../pkgdb/src/buildenv/assets/set-prompt.zsh.sh;
     };
+
+    # Used by `buildenv --container' to create a container builder script.
+    CONTAINER_BUILDER_PATH = builtins.path {
+      name = "mkContainer.nix";
+      path = ../../pkgdb/src/buildenv/assets/mkContainer.nix;
+    };
+
+    # Used by `buildenv --container' to access `dockerTools` at a known version
+    # When utilities from nixpkgs are used by flox at runtime,
+    # they should be
+    # a) bundled at buildtime if possible (binaries/packages)
+    # b) use this version of nixpkgs i.e. (nix library utils such as `dockerTools`)
+    COMMON_NIXPKGS_URL = let
+      lockfile = builtins.fromJSON (builtins.readFile ./../../flake.lock);
+      root = lockfile.nodes.${lockfile.root};
+      nixpkgs = lockfile.nodes.${root.inputs.nixpkgs}.locked;
+    in
+      builtins.flakeRefToString nixpkgs;
```

Comment:
question blocking: this wasn't added till Nix 2.18 (https://nixos.org/manual/nix/stable/release-notes/rl-2.18). Do we want to require flox developers to all bump to 2.18? Especially since we're still using 2.17 in flox itself